external-crates/move/crates/move-compiler/src/shared/mod.rs

```diff
@@ -303,6 +304,7 @@ impl CompilationEnv {
             known_code_filter!(FILTER_UNUSED_MUT_REF, UnusedItem::MutReference),
             known_code_filter!(FILTER_UNUSED_MUT_PARAM, UnusedItem::MutParam),
             known_code_filter!(FILTER_IMPLICIT_CONST_COPY, TypeSafety::ImplicitConstantCopy),
+            known_code_filter!(FILTER_DUPLICATE_ALIAS, Declarations::DuplicateAlias),
```

Comment:
With this new alias, can we stop bailing on use fun usage for macros? If someone wants one explicitly to be in macro scope, they can mark it allowed.

Reply:
What is the case exactly? Not sure I follow (though its early in the day for me)